docs/guide/content/user_guide/index.md

```diff
@@ -35,6 +35,7 @@ Built‑in repository connectors handle the details of each repository’s API,
 3. Familiarize yourself with the navigation bar:
     - **Projects**, **Downloads**, **Uploads**, and a **Repositories** drop-down appear from left to right.
     - The **Explore** link toggles a bar where you can paste a DOI or repository URL.
-    - On the far right you’ll see links to **Open OnDemand** (which returns you to the main dashboard) and **Restart**.
-4. The first time you visit, create project so that you can immediately begin adding downloads.
+    - On the far right you’ll see links to **Open OnDemand** (which returns you to the main dashboard) 
+    - There is also a **Help** dropdown where you can find a link to this **Guide**, the **Sitemap** and **Restart**.
+4. The first time you visit, create a project so that you can immediately begin adding downloads.
```

Comment:
I would update to `visit, create a project and make it active so`

Reply:
fixed